doc/langref.html.in

```diff
@@ -4489,8 +4489,7 @@ comptime {
       <pre>{#syntax#}@cmpxchgStrong(comptime T: type, ptr: *T, expected_value: T, new_value: T, success_order: AtomicOrder, fail_order: AtomicOrder) ?T{#endsyntax#}</pre>
       <p>
       This function performs a strong atomic compare-and-exchange operation, returning {#syntax#}null{#endsyntax#}
-      if the current value is not the given expected value. It's the equivalent of this code,
-      except atomic:
+      if the underlying atomic value was successfully changed, otherwise it returns the current value of the atomic at the time of the operation. It's the equivalent of this code, except atomic:
```

Comment:
Just making it a bit more explicit:

```suggestion
      if the value at {#syntax#}ptr{#endsyntax#} was equal to {#syntax#}expected_value{#endsyntax#}
      at the time of the operation and therefore changed to {#syntax#}new_value{#endsyntax#};
      otherwise, it returns the value at {#syntax#}ptr{#endsyntax#} at the time of the operation.
      It's the equivalent of this code, except atomic:
```

Also, this is not super important, but I don't love the terms "atomic value" and "the atomic". We don't put it that way anywhere else in the lang ref, and I think it only really makes sense if you think of `std.atomic.Value(T)`.